scripts/docker-login.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+auth=$(echo -n "${DOCKER_USERNAME}:${DOCKER_PASSWORD}" | base64)
+
+cat << EOF > /kaniko/.docker/config.json
+{
+  "auths": {
+    "https://index.docker.io/v1/": {
```

Comment:
We'll probably want/need to parameterize this registry as well, right?  (Perhaps can use DOCKER_IMAGE_PREFIX, if defined; otherwise default to the value currently used?)

Reply:
I had been thinking we could just update this script if we want to push to somewhere else, as I don't anticipate that being something that changes often. But I'm also open to parameterizing it.

Reply:
Thinking more... I suppose if one were to fork us, they might conceivably want to push the images they build from their fork somewhere else. I suppose this should indeed be parameterized.